### PR TITLE
Move from TOML to JSON format

### PR DESCRIPTION
**Pull request by xsawyerx**
Handling #106.

### CODE REVIEW

**Comment by gonzus** [comment]
Left one comment, otherwise LGTM.

File: lib/Pakket/Repository/Config.pm [hunk 1]
```diff
@@ -30,8 +29,7 @@ sub retrieve_package_config {
 
     my $config;
     eval {
-        $config = TOML::Parser->new( 'strict_mode' => 1 )
-                              ->parse($config_str);
+        decode_json($config_str);
```
Comment (xsawyerx) [inline lib/Pakket/Repository/Config.pm]: Whoops. Yes! :)

**Comment by xsawyerx** [comment]
Fixed.

**Comment by xsawyerx** [comment]
Rebased and merged!

**Pull request closed without merging**